R/drive_upload.R

```diff
@@ -105,16 +105,26 @@ drive_upload <- function(from = NULL,
     up_id <- proc_res$id
   }
 
-  request <- build_request(
-    endpoint = "drive.files.update.media",
-    params = list(
-      fileId = up_id,
-      uploadType = "media",
-      body = httr::upload_file(path = from,
-                               type = mimeType),
-      fields = "*"
-    )
-  )
+  request <- list(url = httr::modify_url(url = .drive$base_url,
```

Comment:
It sounds like a good idea to do what you need to do over in the discovery document prep, i.e. to create two separate entries in our `endpoints` object. It feels cleaner to do special handling there vs. in these functions.